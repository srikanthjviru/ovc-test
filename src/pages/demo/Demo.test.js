import React from 'react';
import ReactDOM from 'react-dom';
import Demo from './Container';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Demo users={[]} loadUsers={() => null} />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('should match with snapshot', () => {
  const component = renderer.create(
    <Demo users={[]} loadUsers={() => null} />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
