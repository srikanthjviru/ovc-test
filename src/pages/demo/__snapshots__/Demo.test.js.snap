// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match with snapshot 1`] = `
<table
  style={
    Object {
      "margin": "0px auto",
      "maxWidth": 800,
    }
  }
>
  <thead>
    <tr>
      <th
        style={
          Object {
            "textAlign": "left",
          }
        }
      >
        Name
      </th>
      <th
        style={
          Object {
            "textAlign": "left",
          }
        }
      >
        Email
      </th>
      <th
        style={
          Object {
            "textAlign": "left",
          }
        }
      >
        City
      </th>
      <th
        style={
          Object {
            "textAlign": "left",
          }
        }
      >
        Company
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
